Source/MillicastPublisher/Private/Media/SlateWindowVideoCapturer.cpp

```diff
@@ -59,14 +85,29 @@ SlateWindowVideoCapturer::FStreamTrackInterface SlateWindowVideoCapturer::GetTra
 
 void SlateWindowVideoCapturer::OnBackBufferReadyToPresent(SWindow& SlateWindow, const FTexture2DRHIRef& Buffer)
```

Comment:
This callback gives the SWindow for every window in UE! So what I did was add a member and function to set the target window. By default it is the main window of the game/editor.